src/components/oppgaver/Oppgaver.tsx

```diff
@@ -178,33 +177,29 @@ const Oppgaver: React.FC<Props> = ({fiksDigisosId}) => {
         }
     }, [sakUtbetalinger, dokumentasjonkrav, vilkar]);
 
-    useEffect(() => {
-        if (oppgaverError || vilkarError || dokumentasjonkravError) {
-            setLoadingResourscesError(true);
-        }
-    }, [oppgaverError, vilkarError, dokumentasjonkravError, LoadingResourscesError]);
-
     return (
-        <StyledPanel error={+LoadingResourscesError}>
+        <StyledPanel error={+hasError}>
```

Comment:
hva gjør plusen?

Reply:
Fikk følgende feilmelding:
```
Warning: Received `false` for a non-boolean attribute `error`.
If you want to write it to the DOM, pass a string instead: error="false" or error={value.toString()}.
If you used to conditionally omit it with error={condition && value}, pass error={condition ? value : undefined} instead.
```
Jeg tenkte, i steden for å begynne å passere en string, at kanskje det var noen andre måte å løse det på.
 
Det [plussen](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus) her gjør er å konverterer resultatet fra boolean til et number. Om jeg forsto det riktig så er det bare en annen måte å skrive `hasError ? 1 : 0` i dette tilfelle.